waspc/e2e-test/test-outputs/waspBuild-golden/waspBuild/.wasp/build/.waspchecksums

```diff
@@ -0,0 +1,303 @@
+[
+    [
+        [
+            "file",
```

Comment:
if there was a folder, it would be "dir"?

Reply:
Yes!